app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -414,23 +419,43 @@ class MainActivity :
 
     override fun onCreate(savedInstanceState: Bundle?) {
         Timber.d("Main Activity onCreate")
+
         // Changing the theme draws the status and navigation bars as black, unless this is manually set
         WindowCompat.setDecorFitsSystemWindows(window, false)
+
+        val splashScreen = installSplashScreen()
+
         super.onCreate(savedInstanceState)
         theme.setupThemeForConfig(this, resources.configuration)
         enableEdgeToEdge(navigationBarStyle = theme.getNavigationBarStyle(this))
         bottomSheetTag = savedInstanceState?.getString(SAVEDSTATE_BOTTOM_SHEET_TAG)
 
         playbackManager.setNotificationPermissionChecker(this)
 
+        var isReady = false
```

Comment:
```suggestion
        var isReady = settings.hasCompletedOnboarding()
```
Should we only wait if the user hasn't completed the onboarding?